### ISSUE TITLE
How to shorten training time?

### ISSUE THREAD

**Issue by zhouweii234** [hkchengrex/STCN#71]
I have 4 2080Ti and I want to shorten training time. Can you give me same suggestions? Should I change the `batch_size` to 1 / 2 or  do something else? In other code I've run before, a larger batch size means less time, but I'm not sure how to shorten training time with the code use `torch.distributed.launch`.

**Comment by hkchengrex** [comment]
If you are using four GPUs, change `nproc_per_node` to `4` in launch.
To keep the total batch size the same, halve the `batch_size` in `util/hyper_para.py`.
To increase the total batch size (times 2), keep the `batch_size` parameter. Possibly you should also double the learning rate and reduce the number of iterations (and also the `step` parameter that controls learning rate decay).

If you are bottlenecked by data loading, this is not going to help. You can increase `num_workers` if you have enough CPU cores.
